outermysite/polls/imdbExactApi.py

```diff
@@ -0,0 +1,92 @@
+import requests
+import sys
+import traceback
+from bs4 import BeautifulSoup
+
+def getMovieResults(Id):
```

Comment:
Every language usually has some convention on how to name variables and functions. For example, Java uses camel casing. In python, underscores are preferred in python for functions and variable names. Class names use upper camel case.